glutton/conntrack.go

```diff
@@ -0,0 +1,58 @@
+package main
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"github.com/MohammadBilalArif/glutton"
+	"github.com/hectane/go-nonblockingchan"
+	"log"
+	"os/exec"
+	"regexp"
+	"strconv"
+)
+
+const regexpression = `\[(\d+\.\d+)(?:\s+)?\]\s+\[\w+]\s+\w+\s+\w+\s\w+\s+.+?src=(\d+\.\d+\.\d+\.\d+)\s+dst=(\d+\.\d+\.\d+\.\d+)\s+sport=(\d+)\s+dport=(\d+)\s+`
+
+const connBufferSize int = 30000000
+
+func Monitor_Connections(proto string, channel *nbc.NonBlockingChan) {
```

Comment:
This is the recommended way: https://golang.org/doc/effective_go.html#names
I use Atom as editor which has a linter which complains if you mess up your names.